contracts/test/StakeTable.t.sol

```diff
@@ -63,6 +49,22 @@ contract StakeTable_Test is Test {
             blsSig
         );
     }
+}
+
+contract StakeTable_Test is Test, StakeTableCommonTest {
+    event Registered(bytes32, uint64, AbstractStakeTable.StakeType, uint256);
+    event Deposit(bytes32, uint256);
+    event Exit(bytes32, uint64);
```

Comment:
maybe we don't need to re-declare these?
I have example when I try to call `expectEmit`, I can just use the original definition: https://github.com/EspressoSystems/espresso-sequencer/blob/main/contracts/test/LightClient.t.sol#L208-L210

Reply:
Done in aeaf0c0.